llama_stack/providers/utils/kvstore/api.py

```diff
@@ -17,3 +17,5 @@ async def get(self, key: str) -> Optional[str]: ...
     async def delete(self, key: str) -> None: ...
 
     async def range(self, start_key: str, end_key: str) -> List[str]: ...
+
+    async def list(self) -> List[str]: ...
```

Comment:
this seems like an impossible API to serve in practice right? you can't ask a Redis or Postgres to just list all keys? It can take arbitrarily long.